packages/core/core/src/RequestTracker.js

```diff
@@ -1227,6 +1231,36 @@ export default class RequestTracker {
     }
   }
 
+  flushStats() {
+    let requestTypeEntries = {};
+    type RequestTypes = Array<$Keys<typeof requestTypes>>;
+
+    for (let key of (Object.keys(requestTypes): RequestTypes)) {
```

Comment:
/nit you could just create this mapping at the top of the file so it's done once, and it could be more idiomatic using `Object.entries`